src/crepe/api/Event.h

```diff
@@ -88,8 +88,28 @@ class MouseMoveEvent : public Event {
 
 	//! Y-coordinate of the mouse position at the time of the event.
 	int mouse_y = 0;
+
+	// Movement since last event in x
+	int rel_x = 0;
+
+	// Movement since last event in y
+	int rel_y = 0;
 };
 
+/**
+ * \brief Event triggered when the mouse is moved.
+ */
+class MouseScrollEvent : public Event {
+public:
+	//! X-coordinate of the mouse position at the time of the event.
+	int scroll_x = 0;
+
+	//! Y-coordinate of the mouse position at the time of the event.
+	int scroll_y = 0;
+
+	//! scroll direction (-1 = down, 1 = up)
+	int direction = 0;
```

Comment:
Ik ben gewend dat je bij scroll events de scrolafstand krijgt? Is dit hoe de events vanuit SDL komen?